### PR TITLE
Adds support for reading package parameters when a leading module name is present

### PR DESCRIPTION
**Pull request by bbranan**
Required due to the addition of module names as a prefix to property elements in DSpace 6.x

### CODE REVIEW

File: src/main/java/org/dspace/ctask/replicate/AbstractPackagerTask.java [hunk 1]
```diff
@@ -65,20 +69,26 @@ protected PackageParameters loadPackagerParameters(String moduleName)
...
         if(moduleProps!=null)
-        {    
+        {
             //loop through all properties in the config file
             for(String property : moduleProps)
             {
+                //Set propertyName, removing leading module name (if applicable)
+                String propertyName = property;
+                if(propertyName.startsWith(moduleName + ".")) {
+                    propertyName = property.replaceFirst(moduleName + ".", "");
```
Comment (tdonohue) [inline src/main/java/org/dspace/ctask/replicate/AbstractPackagerTask.java]: @bbranan : while I'm sure this code works, it looks a bit odd (and potentially fragile).  It's weird to check for the `moduleName` in `propertyName` but then replace it in `property`.  In other words, this likely only works cause you set them equal to each other. If that ever changes, this code would break.  Probably best to use the same variable for both the `startsWith` and the `replaceFirst`.

Long story short, might want to change this to ` propertyName = propertyName.replaceFirst(moduleName + ".", "");`

Comment (bbranan) [reply src/main/java/org/dspace/ctask/replicate/AbstractPackagerTask.java]: 👍
